### PR TITLE
Fix employee request invitation address

### PR DESCRIPTION
**Pull request by MBatrak**

### CODE REVIEW

File: lib/ehealth/utils/address_merger.ex [hunk 1]
```diff
@@ -0,0 +1,137 @@
+defmodule EHealth.Utils.AddressMerger do
+  @moduledoc """
+  Address merger
+  """
+
+  alias EHealth.Dictionaries
+
+  defp config do
+    [{_pid, val}] = Registry.lookup(EHealth.Registry, :address_merger)
```
Comment (gmile) [inline lib/ehealth/utils/address_merger.ex]: @MBatrak is there a reason to use `Registry` here? It's possible to read config during module compilation time by reading config into a module attribute

Comment (gmile) [inline lib/ehealth/utils/address_merger.ex]: @PavelVesnin what do you think?

Comment (PavelVesnin) [inline lib/ehealth/utils/address_merger.ex]: @gmile it's nice idea about module attributes. I think we can try it

Comment (MBatrak) [inline lib/ehealth/utils/address_merger.ex]: Done

**Pull request merged**